Dockerfile

```diff
@@ -40,27 +40,38 @@ FROM builder as testing
 
 WORKDIR /workspace
 
-RUN apk add rsync curl tar
+RUN apk add curl tar
 
 # These two steps follow the Kubebuilder envtest playbook https://book.kubebuilder.io/reference/envtest.html
 ARG K8S_VERSION=1.23.3
 RUN curl -sSLo envtest-bins.tar.gz "https://go.kubebuilder.io/test-tools/${K8S_VERSION}/$(go env GOOS)/$(go env GOARCH)" \
     && mkdir /usr/local/kubebuilder \
     && tar -C /usr/local/kubebuilder --strip-components=1 -zvxf envtest-bins.tar.gz
 
-
 COPY config/ config/
 COPY initiateContainerTest.sh .
 
 ENTRYPOINT [ "sh", "initiateContainerTest.sh" ]
 
 ###############################################################################
-FROM alpine:latest
+FROM ghcr.io/nearnodeflash/nnf-mfu:latest
+
+RUN apt update
+
+RUN apt install -y openmpi-bin
+
+# TODO Remove this
+RUN apt install -y bash
 
-RUN apk add rsync
+# The following lines are from the mpiFileUtils (nnf-mfu) Dockerfile; 
+# do not change them unless you know what it is you are doing
+ARG port=2222
```

Comment:
If this is already in nnf-mfu, then you don't need to repeat it here.    The above FROM means we're in that container now.

Reply:
See my response to [Blake's question ](https://github.com/NearNodeFlash/nnf-dm/pull/37#discussion_r948422900)